### PR TITLE
DOC: Fixing EX01 - Added examples

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

Towards https://github.com/pandas-dev/pandas/issues/37875

I could not make the html of each method, so I'm not sure if everything looks well.
I'll use `/preview` here.

### CODE REVIEW

File: pandas/core/window/rolling.py [hunk 1]
```diff
@@ -1270,7 +1270,32 @@ def aggregate(self, func, *args, **kwargs):
...
+            """
+        >>> ser = pd.Series([0, 1, 5, 2, 8])
+
+        In order to get an instance of pandas.core.window.rolling.Window we need
+        to pass the parameter `win_type`.
+
+        >>> type(ser.rolling(2, win_type='gaussian'))
+        <class 'pandas.core.window.rolling.Window'>
+
+        In order to use the `Sci-py` gaussian window we need to provide the parameters
```
Comment (MarcoGorelli) [inline pandas/core/window/rolling.py:1285]: I think we can just write "SciPy"

also, gaussian => Gaussian

File: pandas/core/window/rolling.py [hunk 2]
```diff
@@ -1338,7 +1411,31 @@ def var(self, ddof: int = 1, numeric_only: bool = False, **kwargs):
...
+
+        >>> ser.rolling(2, win_type='gaussian').std(std=3)
+        0         NaN
+        1    0.707107
+        2    2.828427
+        3    2.121320
+        4    4.242641
+        dtype: float64
+        """
+        ).replace("\n", "", 1),
```
[the remark this answers is not in the document]
Comment (DeaMariaLeon) [reply pandas/core/window/rolling.py:1438]: Ouch... I just saw this one! - I'll change it.

File: pandas/core/window/rolling.py [hunk 3]
```diff
@@ -1270,7 +1270,32 @@ def aggregate(self, func, *args, **kwargs):
...
+        >>> ser = pd.Series([0, 1, 5, 2, 8])
+
+        In order to get an instance of pandas.core.window.rolling.Window we need
+        to pass the parameter `win_type`.
+
+        >>> type(ser.rolling(2, win_type='gaussian'))
+        <class 'pandas.core.window.rolling.Window'>
+
+        In order to use the `Sci-py` gaussian window we need to provide the parameters
+        M and std. The parameter M corresponds to 2 in our example.
```
[the remark this answers is not in the document]
Comment (DeaMariaLeon) [reply pandas/core/window/rolling.py]: Sorry, I forgot about that.

File: pandas/core/window/rolling.py (hunk 1 above)
Comment (DeaMariaLeon) [reply pandas/core/window/rolling.py]: Done

File: pandas/core/window/rolling.py [hunk 4]
```diff
@@ -1270,7 +1270,32 @@ def aggregate(self, func, *args, **kwargs):
...
         template_returns,
         create_section_header("See Also"),
-        template_see_also[:-1],
+        template_see_also,
+        create_section_header("Examples"),
+        dedent(
+            """\
+        >>> ser = pd.Series([0, 1, 5, 2, 8])
+
+        To get an instance of :class: ~pandas.core.window.rolling.Window we need
```
Comment (MarcoGorelli) [inline pandas/core/window/rolling.py:1279]: I think it might need to be
```
:class:`~pandas.core.window.rolling.Window`
```

Comment (DeaMariaLeon) [reply pandas/core/window/rolling.py]: Done - thanks.

**Comment by MarcoGorelli** [comment]
Nice! Looks like there's just a merge conflict to resolve

**Review by MarcoGorelli — APPROVED** [review]
nice, should be good, thanks @DeaMariaLeon !

**Pull request merged**